.github/workflows/release-please.yaml

```diff
@@ -77,13 +78,20 @@ jobs:
             VERSION=${{ needs.release-please.outputs.release_tag_name }}
             COMMIT=${{ github.sha }}
             DATE=${{ steps.date.outputs.date }}
+        outputs:
+          image_digest: ${{ steps.build.outputs.digest }}
 
+  container-signing:
+    needs: container-release
+    runs-on: ubuntu-latest
+    if: ${{ needs.release-please.outputs.release_created }}
+    steps:
       - name: Install Cosign
         uses: sigstore/cosign-installer@main
```

Comment:
Can you explicitly set the version or commit sha here please?

Reply:
^

Reply:
I vote 2

Reply:
I updated gh action to use digest instead of @main